app/models/live_metric.rb

```diff
@@ -0,0 +1,112 @@
+class LiveMetric < ActsAsArModel
+  set_columns_hash(:timestamp => :datetime)
+
+  class LiveMetricError < RuntimeError; end
+
+  def self.where(*)
+    raise LiveMetricError, "where method is not supported on LiveMetric"
```

Comment:
Probably this method is not necessary.
I've started implemented the needed methods for the parent class and after that I didn't check if this where could be removed.
I'm going to review it and clean it if logic works with the parent class 'where' method.